### PR TITLE
Release v1.3.0

### PR DESCRIPTION
**Pull request by brunobord**
## Release

* [x] Fetch translations from Crowdin
* [x] Change VERSION with the appropriate tag
* [x] Amend `CHANGELOG.rst` (check the release date)
* [ ] Tag the resulting commit with the appropriate tag
* [ ] Push the tag (using: `git push --tags`)
* [ ] Edit the release (copy/paste CHANGELOG)
* [ ] Publish the new release to PyPI

### CODE REVIEW

**Comment by brunobord** [comment]
Note to reviewers: this PR includes the proper release **and** the "back to dev" commit.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,8 +5,16 @@ ChangeLog
 master (unreleased)
 ===================
 
-- Allow POST method for form validation endpoint. This version is the last to support the GET verb for the validation endpoint. We'll drop it in the next version.
+Nothing here yet.
+
+Realease 1.3.0 (2018-02-14)
```
Comment (syldlb) [inline CHANGELOG.rst]: :smiley:

Comment (brunobord) [reply CHANGELOG.rst]: ooops typo. fixed.

Comment (syldlb) [reply CHANGELOG.rst]: Wow, I missed it and commented for no reason :thinking:

**Pull request merged**